File: metawish.c
#include<stdio.h>
#include<string.h>

main()
{
    FILE *ifp;
    //open file
    ifp = fopen("metawish.in","r");
    int T,t;
    fscanf(ifp,"%d",&T);
    //create a character array to store input string
    char string[10000];
    fgets(string,sizeof string,ifp);
    //for each testcase
    for(t=1;t<=T;t++){
        fgets(string,sizeof string,ifp);
        int a,ok = 0;
        //start checking after "I wish "..
        for(a = 7;a<strlen(string);a++){
        
            //check for wish, Wish, WIsh, .... WISH
            if((string[a-3]=='w'||string[a-3]=='W')
                &&(string[a-2]=='i'||string[a-2]=='I')
                &&(string[a-1]=='s'||string[a-1]=='S')
                &&(string[a]=='h'||string[a]=='H'))
                    ok=1;
        }
        if(ok==0)
            printf("Wish Response #%d: Your wish will be granted.\n",t);
        else
            printf("Wish Response #%d: Stop trying to wish for more wishes!\n",t);

    }

}

File: metawish.java
import java.util.*;
import java.io.*;

public class metawish
{
   public static void main(String[] args) throws IOException
   {
      // Creates a scanner to read from stdin.
      Scanner in = new Scanner(System.in);
      // Reads the number of test cases.
      int t = in.nextInt();
      // Reads the empty rest of line after the number of test cases.
      in.nextLine();
      // Loops through each test case.
      // Each input will start with "I wish for" so we consider the substring after the 10th character.
      // The capitalization does not matter, so we make the rest of the string lower case.
      // Then we check if the rest of string contains the string "wish"
      // If it doesn't the wish will be granted, else it won't be.
      for (int i = 1; i <= t; i++)
         if (!(in.nextLine()).substring(10).toLowerCase().contains("wish"))
            System.out.println("Wish Response #" + i + ": Your wish will be granted.");
         else
            System.out.println("Wish Response #" + i + ": Stop trying to wish for more wishes!");
   }
}
